source/_patterns/00-atoms/components/orcid.mustache

```diff
@@ -1,10 +1,9 @@
 <span class="orcid">
-  <a href="https://orcid.org/">
+  <a href="https://orcid.org/{{id}}">
     <picture>
       <source srcset="{{#assetRewrite}}{{assetsPath}}/img/icons/orcid.svg{{/assetRewrite}}" type="image/svg+xml">
       <img src="{{#assetRewrite}}{{assetsPath}}/img/icons/orcid.png{{/assetRewrite}}" class="orcid__icon"
            alt="ORCID icon">
-    </picture>
-  </a>
-  <a href="https://orcid.org/{{id}}" class="orcid__id">{{id}}</a>
-</span>
+    </picture> <span class="visuallyhidden">"This ORCID iD identifies the author of this article:"</span>
```

Comment:
Don't need the extra `iD` as the "ID" in `ORCID` has it covered.

Reply:
Ignore this comment. It appears ORCID use this tautology themselves, so we should too. Sorry for the confusion.